README.md

```diff
@@ -69,17 +73,16 @@ fastify.get("/sendmail", async (req, reply) => {
 
 #### Templates
 Each message must have the following templates with the file extension set in point-of-view config:
-  - `from`: Contains email address the email is to be sent from.
-  - `subject`: Contains the subject for the email.
-  - `html`: Contains the html for the email.
+  - `html`: Contains the html template for the email.
+  - `text`: Contains the text template for the email.
+
 ```
 .
 |--index.js
 |--templates
    |-- email
```

Comment:
if we've simplified and removed the from and subject do we still need the `email` nesting here?  /templates should be good enough and have html and text inside them?

Reply:
yep